Framework/Kernel/CMakeLists.txt

```diff
@@ -659,7 +659,7 @@ install ( TARGETS Kernel ${SYSTEM_PACKAGE_TARGET} DESTINATION ${LIB_DIR} )
 
 # Create the properties file for the installer
 set ( MANTID_ROOT_BUILD ${MANTID_ROOT} )
-if ( APPLE )
+if ( APPLE AND ENABLE_MANTIDPLOT )
```

Comment:
Is `ENABLE_MANTIDPLOT` the correct test here or do we need another flag like `CONDA_BUILD` or something? I'm just thinking about when we remove MantidPlot...